plugins/modules/wafv2_web_acl.py

```diff
@@ -137,10 +143,56 @@
           managed_rule_group_statement:
             vendor_name: AWS
             name: AWSManagedRulesAdminProtectionRuleSet
+    purge_rules: yes
     tags:
       A: B
       C: D
-  register: out
+    state: present
+
+- name: Create IP filtering web ACL
+  community.aws.wafv2_web_acl:
+    name: ip-filtering-traffic
+    description: ACL that filters web traffic based on rate limits and whitelists some IPs
+    scope: REGIONAL
+    default_action: Allow
+    sampled_requests: yes
+    cloudwatch_metrics: yes
+    metric_name: ip-filtering-traffic
+    rules:
+      - name: whitelist-own-IPs
+        priority: 0
+        action:
+          allow: {}
+        statement:
+          ip_set_reference_statement:
+            arn: 'arn:aws:wafv2:us-east-1:520789123123:regional/ipset/own-public-ips/1c4bdfc4-0f77-3b23-5222-123123123'
+        visibility_config:
+          sampled_requests_enabled: yes
+          cloud_watch_metrics_enabled: yes
+          metric_name: waf-acl-rule-whitelist-own-IPs
+      - name: rate-limit-per-IP
+        priority: 1
+        action:
+          block:
+            custom_response:
+              response_code: 429
+              custom_response_body_key: too_many_requests
+        statement:
+          rate_based_statement:
+            limit: 5000
+            aggregate_key_type: IP
+        visibility_config:
+          sampled_requests_enabled: yes
+          cloud_watch_metrics_enabled: yes
+          metric_name: waf-acl-rule-rate-limit-per-IP
+        purge_rules: yes
+    custom_response_bodies:
+      too_many_requests:
+        ContentType: APPLICATION_JSON
```

Comment:
the example shows `content_type` and `content`

Reply:
I guess you are referring to the RETURN examples. Yes those go through the snake case converter hence looking different then the input examples. I guess it's a bit confusing and I should probably also use snake case as input format here (as you already suggested in other comment).